src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Management.Automation;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text;
+using OutGridView.Models;
+using Terminal.Gui;
+
+namespace OutGridView.Cmdlet
+{
+    internal class ConsoleGui
+    {
+        private const string ACCEPT_TEXT = "Are you sure you want to select\nthese items to send down the pipeline?";
```

Comment:
Doesn't have to be part of this PR, but this text should be in a resx for localization purposes.